pkgs/development/libraries/freeimage/default.nix

```diff
@@ -16,7 +16,7 @@ stdenv.mkDerivation {
 
   # Ensure that the bundled libraries are not used at all
   prePatch = "rm -rf Source/Lib* Source/OpenEXR Source/ZLib";
```

Comment:
```suggestion
  postPatch = ''
    rm -rf Source/Lib* Source/OpenEXR Source/ZLib
  '';
```

Reply:
I changed this back to `prePatch` because this failed since there's already a `postPatch`. I don't know enough about what these were intended to do to know if they should be merged into one postPatch.